### PR TITLE
add .gitignore files

### PR DESCRIPTION
**Pull request by uli42**
SSIA

### CODE REVIEW

File: nx-X11/lib/X11/.gitignore
Comment (sunweaver) [inline nx-X11/lib/X11/.gitignore]: This can go away anyway, after #279 has been merged...

File: nx-X11/.gitignore
Comment (sunweaver) [inline nx-X11/.gitignore:4]: What about .build-exports?

**Review by sunweaver — CHANGES_REQUESTED** [review]
Please consider my comments, I will merge this once we have xlibi18n removed.

File: nx-X11/.gitignore
Comment (uli42) [reply nx-X11/.gitignore:4]: I neither know nor have that!?

File: nx-X11/lib/X11/.gitignore
Comment (uli42) [reply nx-X11/lib/X11/.gitignore]: sure

**Comment by sunweaver** [comment]
Hi Uli,

On  Fr 04 Nov 2016 11:48:43 CET, Ulrich Sibiller wrote:

> uli42 commented on this pull request.
> 
> > @@ -0,0 +1,4 @@
> > +Makefile
> > +Makefile.bak
> > +xmakefile
> > +exports/
> 
> I neither know nor have that!?

the nx-X11/.build-exports gets created when running make install on  
our main Makefile. It gets called whenever you build the nx-libs code  
with dpkg-buildpackage.

## Mike

DAS-NETZWERKTEAM
mike gabriel, herweg 7, 24357 fleckeby
mobile: +49 (1520) 1976 148
landline: +49 (4354) 8390 139

GnuPG Fingerprint: 9BFB AEE8 6C0A A5FF BF22  0782 9AF4 6B30 2577 1B31
mail: mike.gabriel@das-netzwerkteam.de, http://das-netzwerkteam.de

**Comment by uli42** [comment]
OK, have added `.build-exports/`

**Comment by uli42** [comment]
rebased, but should be adapted as soon as #279 has been merged

**Comment by sunweaver** [comment]
@uli42: #279  has been merged, please make this PR ready for merging it finally. Thanks.

**Comment by uli42** [comment]
rebased + removed xlibi18n stuff from .gitignore

**Review by sunweaver — APPROVED** [review]

**Pull request merged**
